src/outputs/parthenon_hdf5.cpp

```diff
@@ -310,13 +310,13 @@ void PHDF5Output::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm) {
       // open vector space
       // write mesh block size
       const hsize_t xDims[1] = {3};
-      int meshblock_size[xDims[0]] = {nx1, nx2, nx3};
+      int meshblock_size[3] = {nx1, nx2, nx3};
       writeH5AI32("MeshBlockSize", meshblock_size,
                   H5S::FromHIDCheck(H5Screate_simple(1, xDims, NULL)), infoDSet);
 
       // RootGridDomain - float[9] array with xyz mins, maxs, rats (dx(i)/dx(i-1))
       const hsize_t rootGridDomain_size[1] = {9};
-      Real rootGridDomain[rootGridDomain_size[0]] = {
+      Real rootGridDomain[9] = {
```

Comment:
I prefer to have named constants that are used in place of magic numbers even when it seems obvious, but this is more preference.

Reply:
I generally agree. Here I'm not sure I could come up with a good name and the number is only used in two adjacent lines for defining the array dims so I'm leaning towards keeping it as they are here.

Reply:
My vote would be don't put a number here at all. Just let the compiler figure it out.